modules/renter/contractor/uptime.go

```diff
@@ -0,0 +1,94 @@
+package contractor
+
+import (
+	"time"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/modules"
+)
+
+// uptimeInterval specifies how frequently hosts are checked.
+var (
+	uptimeInterval = func() time.Duration {
+		switch build.Release {
+		case "dev":
+			return 1 * time.Minute
+		case "standard":
+			return 30 * time.Minute
+		case "testing":
+			return 100 * time.Millisecond
+		}
+		panic("undefined uptimeInterval")
+	}()
+
+	// uptimeWindow specifies the duration in which host uptime is checked.
+	uptimeWindow = func() time.Duration {
+		switch build.Release {
+		case "dev":
+			return 24 * time.Hour // 1 day
+		case "standard":
+			return 7 * 24 * time.Hour // 1 week
+		case "testing":
+			return 1 * time.Minute // 1 minute
```

Comment:
same here, a minute is a really long time when running automated tests. Perhaps 15 seconds?